destinations/service_test.go

```diff
@@ -108,7 +109,7 @@ func TestServiceInit(t *testing.T) {
 	payload := &payloadHolder{payload: []byte(initialDestinations)}
 	mockDestinationsServer := startTestServer(payload)
 
-	service, err := NewService(context.Background(), nil, mockDestinationsServer.URL, "/tmp", "/tmp/fallback", 5, nil, createTestStorage)
+	service, err := NewService(context.Background(), nil, mockDestinationsServer.URL, "/tmp", "/tmp/fallback", 5, nil, nil, createTestStorage)
```

Comment:
it would be better to use here empty logger instead of nil

Reply:
and in other places

Reply:
It's not queryLogger here, it's io.Writer. In fact, it may be nil if SQL debug logs are disabled. So, I think that here it may be nil.

Reply:
I've fixed queryLogger nil -> &QueryLogger{} at other places at tests